examples/server/server.c

```diff
@@ -59,18 +59,19 @@
  * by the client. */
 
 static const char webServerMsg[] =
-    "HTTP/1.1 200 OK\n"
-    "Content-Type: text/html\n"
-    "Connection: close\n"
-    "\n"
-    "<html>\n"
-    "<head>\n"
-    "<title>Welcome to wolfSSL!</title>\n"
-    "</head>\n"
-    "<body>\n"
-    "<p>wolfSSL has successfully performed handshake!</p>\n"
-    "</body>\n"
-    "</html>\n";
+    "HTTP/1.1 200 OK\r\n"
+    "Content-Type: text/html\r\n"
+    "Connection: close\r\n"
+    "Content-Length: 215\r\n"
```

Comment:
Isn’t the content length different now with the extra carriage return characters?